tests/DataFactories/Newsletter.php

```diff
@@ -213,6 +219,16 @@ public function withSendingQueue(array $options = []) {
     return $this;
   }
 
+  public function beingSentToSubscriber($subscriber, array $data = []) {
```

Comment:
I am not sure about the name. All other methods in the data factories start with `with`. I think this shouldn't be an exception...

Reply:
I will rename it to `withSubscriber`.